go.work

```diff
@@ -0,0 +1,3 @@
+go 1.17
```

Comment:
go.work shouldn't be committed. We should add it to .gitignore

Reply:
Woops, I was playing around with it and I forgot it was there.